### ISSUE TITLE
State with negative activation in all areas

### ISSUE THREAD

**Issue by lsyoath** [OHBA-analysis/HMM-MAR#95]
Hello, 

Thank you for your attention. 

By get_mean function, we can generate activation map for each state, and regions with higher values are tend to be more activated. 

But I observed one state with all regions showing negative activation values. 
What dose this mean ?  Does this imply the state is  "not good" ? 

Looking forward to your reply.

Best wishes,
Siya

**Comment by lsyoath** [comment]
Hello, 

There is also a small problem about parameter setting.

We notice that we need to set TR before we run HMM, and different TR values lead to different activation patterns of states.

Here I want to ask dose the TR value have to be consistent with the acquisition parameters?

Thank you again.

Best, 
Siya

**Comment by vidaurre** [comment]
Hi, 

Regarding the first question, it simply reflects a state of general depression of activity throughout the brain. This is not wrong, it's in the data –and general down states are seen often in the literature. 

I am a bit confused about the second question. There is no parameter TR in the toolbox. Do you mean that different acquisition sequences in the fMRI scanner lead to different activation patterns of states? That wouldn't be entirely surprising. 

Best,
d

**Comment by lsyoath** [comment]
Hi,
Thank you for your explanation about the first question. I totally understand now.

About the second question, I've seen in an example named 'PNAS2017_fullpipeline.m', in line 33 (options.Fs = 1/TR; ).
I thought it was a parameter that had to be determined before running the function hmmmar, and by coincidence, I discovered that changing the value of TR resulted in a completely different state classification result. Therefore, I raised the second question.

Sorry, this may be an immature question, because I don't quite understand the exact principle of the HMM-MAR algorithm.

 Thank you sincerely for your answer, and I look forward to your next reply!

Best,
Siya
